api/config/cdr_config_local.json

```diff
@@ -0,0 +1,79 @@
+{
+  "accessTiers": [
+    {
+      "accessTierId": 1,
+      "shortName": "registered",
+      "displayName": "Registered Tier",
+      "servicePerimeter": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test",
```

Comment:
These values are copied from the existing values in config_*.json

Reply:
Yes, https://precisionmedicineinitiative.atlassian.net/browse/RW-6190

Good catch.  I added a doc update step to 6190.